### PR TITLE
Add how-to to sidebar

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Need to expose how-to articles to public
- Removing one outdated topic (content was moved ages ago) and added redirect and tested locally.

### Resolves:
 [OKTA-155839](https://oktainc.atlassian.net/browse/OKTA-155839)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
replaced by 1828

**Comment by mystiberry-okta** [comment]
how-to2
how-to1
@johngronberg-okta  Here's what the table of contents and splash page for docs will look like. Is that last topic still valid? Do any of the topics need to be re-ordered under "How-To"?

**Comment by mystiberry-okta** [comment]
1. The troubleshooting content was moved here ages ago (and there's a redirect): https://developer.okta.com/docs/api/resources/oidc#troubleshooting
2. We should maybe chat about fixing the diaspora of API AM topics and how best to fix. For now, I think since most of the best practices are things to think about before you start coding, it's in a good place. But we definitely need to pull all these API AM threads together...
3. Will look at the titles and give them a good think.

**Comment by mystiberry-okta** [comment]
@cwu-okta says we'll always need Upgrade SAML Apps to SHA 256, fyi.

**Comment by mystiberry-okta** [comment]
@johngronberg-okta We have to keep the titles a bit short, but I've tried to give proper context and a sense of the goal. Feel free to edit or request any changes. 
v2-how-to-titles

File: _source/_docs/how-to/byo_saml.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
 ---
 layout: docs_page
-title: Bring Your Own SAML App Certificate
+title: Upload Your Own SAML Certificates for Outbound SAML Apps
 excerpt: How to use a custom SAML certificate for apps
 ---
 
-<a name="top"></a>
-## Bring Your Own SAML App Certificate
+# Upload Your Own SAML Certificates for Outbound SAML Apps
```
Comment (johngronberg-okta) [inline _source/_docs/how-to/byo_saml.md]: Can we say "Upload your own Certificates for SAML Apps"?

**Pull request merged**